mopidy/core/playlists.py

```diff
@@ -182,7 +182,7 @@ def delete(self, uri):
 
         with _backend_error_handling(backend):
             backend.playlists.delete(uri).get()
-            # TODO: emit playlist changed?
+            listener.CoreListener.send('playlist_changed', playlist=None)
```

Comment:
A `playlist_deleted` event has now been added in the develop branch. Please rebase or merge `develop` and update this PR.